src/ngraph/runtime/cpu/mkldnn_utils.cpp

```diff
@@ -98,11 +98,20 @@ static const std::map<memory::format, const std::string> s_mkldnn_format_string_
     {memory::format::chwn, "memory::format::chwn"},
     {memory::format::nChw8c, "memory::format::nChw8c"},
     {memory::format::nChw16c, "memory::format::nChw16c"},
+    {memory::format::ncdhw, "memory::format::ndhwc"},
+    {memory::format::ncdhw, "memory::format::ndhwc"},
+    {memory::format::nCdhw16c, "memory::format::nCdhw16c"},
     {memory::format::oi, "memory::format::oi"},
     {memory::format::io, "memory::format::io"},
     {memory::format::oihw, "memory::format::oihw"},
     {memory::format::ihwo, "memory::format::ihwo"},
     {memory::format::hwio, "memory::format::hwio"},
+    //{memory::format::dhwio, "memory::format::dhwio"},
```

Comment:
why is this commented out?

Reply:
dhwio still not in latest mkl-dnn release. Added a TODO there for me.